jans-cedarling/cedarling/examples/tokens.example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why am I still seeing raw binary tokens?

Reply:
that's the example file to show how the user can use their own tokens from the test server. please see [authorize_with_validation.rs](https://github.com/JanssenProject/jans/blob/7ec98e97bd72d2e91de8828266de142179aafa74/jans-cedarling/cedarling/examples/authorize_with_jwt_validation.rs) for the instructions.